src/main.js

```diff
@@ -200,6 +201,14 @@ app.whenReady().then(async () => {
 		createHelpWindow(mainWindow)
 	})
 
+	ipcMain.handle('upgrade:show', () => {
+		const upgradeWindow = createUpgradeWindow()
+		createMainWindow = upgradeWindow
```

Comment:
**issue (blocking):** `createMainWindow` must store the function for re-creating the window, not the window itself

```suggestion
		createMainWindow = createUpgradeWindow
```